sdk/textanalytics/ai-text-analytics/samples/typescript/src/analyzeSentiment.ts

```diff
@@ -2,31 +2,47 @@
 // Licensed under the MIT License.
 
 /**
- * analyzes the sentiment of a piece of text
+ * Demonstrates how to analyze sentiment in documents.
+ * An overall and per-sentence sentiment is returned.
  */
 
-import {
-  TextAnalyticsClient,
-  AzureKeyCredential
-} from "@azure/ai-text-analytics";
+import { TextAnalyticsClient, AzureKeyCredential } from "@azure/ai-text-analytics";
```

Comment:
This is more curiosity, but how does this import differentiate between the ts client and js client?